service/src/main/java/skills/auth/userCommunity/UserCommunityAuthorizationManager.groovy

```diff
@@ -147,6 +166,19 @@ class UserCommunityAuthorizationManager implements AuthorizationManager<RequestA
         return StringUtils.EMPTY
     }
 
+    private String extractQuizId(HttpServletRequest request) {
+        String res = StringUtils.EMPTY
+        String url = getRequestUrl(request)
+        Matcher gid = QUIZ_ID.matcher(url)
+        if (gid.matches()) {
+            res = gid.group(1)
+            if (res?.equalsIgnoreCase("null") ) {
```

Comment:
why do we need this check for "null"?  if necessary, wouldn't the other extractXXXId methods also need this check?